### DISCUSSION TITLE
HTML status file

### DISCUSSION THREAD

**Discussion by DominikSchaller** [jamulussoftware/jamulus#1493] [Technical Support & Help] [answered]
How does the HTML status file work? I didn't find any good explanation, just this:

-m | --htmlstatus | Enable HTML status file, set path and file name | (server only)
-- | -- | -- | --

**Comment by hoffie** [comment, marked as answer]
There is not much more to it. You just start your server with `Jamulus -s --htmlstatus /tmp/status.html` (for example) and Jamulus will update the status file with a list of connected client names.

**Reply by DominikSchaller** [reply]
I'm an HTML noob. What do I need to do to be able to access the html page from an outside browser?

**Reply by hoffie** [reply]
Ah, that's more about exposing this file via HTTP. Jamulus does not contain a HTTP server. All it does is writing this data to a local file. I don't think it's even a full HTML page, but rather just a HTML list. Browsers will probably render it OK, but it's not how HTML is supposed to be used. I think the idea is that this file can be read by dynamic web applications so that this data can be embedded into websites.

But back to your question: You will need a web server such as Apache httpd, nginx or something. What operating system are you using on your server? All common Linux distributions should include a web server in their package repositories. After installing and starting/enabling they will usually start exposing a pre-defined directory such as /srv/www or /var/www, for example.

**Reply by DominikSchaller** [reply]
Thank you! I'm using Linux, learned a lot the last few months. I believe the remote recording feature is done in a similar way (using a web server), I will take a look there.

**Reply by ann0see** [reply]
If I were you, I'd use something like softins' Jamulus-PHP: https://github.com/softins/jamulus-php

**Reply by DominikSchaller** [reply]
Ah, perfect, I'll take a look at that as well.
